harlog.go

```diff
@@ -124,10 +135,23 @@ func (ctx *ProxyCtx) LogToHARFile(captureContent bool) Next {
 	return NEXT
 }
 
+func (ctx *ProxyCtx) GetHar(filename string) {
+	ctx.proxy.FlushHARToDisk(filename)
+}
+
 func (ctx *ProxyCtx) FlushHARToDisk(filename string) {
 	ctx.proxy.FlushHARToDisk(filename)
 }
 
+func (proxy *ProxyHttpServer) FlushHARMemory() (chan *har.Har){
+	proxy.Logf("Calling a flush of HAR to memory")
+	harMemory := make(chan *har.Har)
+
+	proxy.HarFlushToMemory <- harMemory
```

Comment:
I am using the HAR information to use decide when to close a browser after that last request

```go
select {
case <-timer.C:
    browser.Stop()
    isDone = true
case err = <-browser.done:
    if err != nil {
        fmt.Printf("process done with error = %v", err)
    }
    isDone = true
case harOutput = <-proxyHarChan:
    if harOutput != nil {
        numRequests := len(harOutput.Log.Entries)
        currentRequestTime := harOutput.Log.Entries[numRequests-1].StartedDateTime
        if currentRequestTime != lastRequestTime {
            timer.Reset(time.Second * m.TimeOut)
            lastRequestTime = currentRequestTime
        }
    }
    continue
}
```